src/freenet/node/MasterKeys.java

```diff
@@ -273,14 +269,8 @@ private void write(File masterKeysFile, String newPassword, Random hardRandom) t
 		byte[] salt = new byte[32];
 		hardRandom.nextBytes(salt);
 
-        byte[] pwd;
-        try {
-            pwd = newPassword.getBytes("UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            // Impossible
-            throw new Error(e);
-        }
-        MessageDigest md = SHA256.getMessageDigest();
+		byte[] pwd = newPassword.getBytes(StandardCharsets.UTF_8);
+		MessageDigest md = SHA256.getMessageDigest();
```

Comment:
Yeah, fixed that.
There is a mixture of tabs and spaces in quite many files.
I fixed only places mentioned here, and fixing everything else will require a larger effort.

What do you think of common [.editorconfig](https://editorconfig.org), which may help to solve such issues in the future?
I may submit such PR with some initial config,
but enforcing those rules for a whole project needs an administrative decision, 
because there will be quite many changes and it will affect the git history and authorship of the changes.

Reply:
I personally am totally not a friend of such things. Yes, we should have a style guide (that we enforce in code reviews) but existing code should be treated on a case-by-case basis, i.e. match the context of the surrounding code and if convenient (i.e. when adding new code) stick to the style guide. But I don’t even know if we _have_ a style guide. 🙂

Reply:
ok, for upcoming changes i will try to use same rules as are in the file being edited.

Reply:
Yeah, sorry about that… that’s always the trouble with projects that are 20 year old and have been started in an environment where you could write your code whatever way you wanted… Go actually did this one thing correctly, they included a formatter with the compiler so that all Go source code looks the same. 🙂